src/fitnesseMain/FitNesseMain.java

```diff
@@ -49,6 +51,30 @@ public static void main(String[] args) {
   protected static void exit(int exitCode) {
     System.exit(exitCode);
   }
+  
+  // Testing code anaylsis tools
+  public String testMethod(){
+    
+  }
+  
+  public String badCommaWhiteSpaceBeforeAndAfter(String input ,String anotherInput)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') ',' is not followed by whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.cloudmok.org/coding_rules#rule_key=checkstyle%3AWhite_Space_After_Custom)